src/web/modules/gateway_accounts/switch_psp/switch_psp.http.ts

```diff
@@ -6,6 +6,19 @@ import logger from "../../../../lib/logger";
 
 import stripeTestAccount from '../../stripe/test-account.http'
 
+async function setConnectorStripeSetup(gatewayAccountId: string, isTest: boolean) {
```

Comment:
Given we are removing sometimes, can this be named to something like `updateOrRemoveConnectorStripeSetupTasks`

Reply:
this is still a patch operation so remove felt like the wrong verb

Reply:
I would assume they are being set to true/false for a PATCH, but we are removing things in Connector. It should be fine to make it clear.